RUMM-3470: Update documentation for Android SDK v2 (#19062)
DataDog/documentation: content/en/real_user_monitoring/android/troubleshooting.md

 
This means that even if users open your application while offline, no data is lost. To ensure the SDK does not use too much disk space, the data on the disk is automatically discarded if it gets too old.

## Migrating to 2.0.0

> If you've been using the SDK v1, there are some breaking changes introduced in version `2.0.0`. Check our [migration guide][1] to get more information.

Reviewer: ```suggestion
If you've been using the SDK v1, there are some breaking changes introduced in version `2.0.0`. See the [migration guide][1] for more information.
```
